addon/components/ilios-calendar-event-month.js

```diff
@@ -38,6 +38,18 @@ export default CalendarEvent.extend({
   isIlm: notEmpty('event.ilmSession'),
   isOffering: notEmpty('event.offering'),
   clickable: any('isIlm', 'isOffering'),
+
+  recentlyUpdated: computed('event', {
+    get() {
+      const event = this.event;
```

Comment:
I think we can directed access property without Ember getters now, especially ones that are not CPs and promises.